gpu/src/lib.rs

```diff
@@ -136,6 +172,74 @@ pub struct ColorAttr {
   pub mask_head: i32,
 }
 
+#[repr(packed)]
+#[derive(AsBytes, PartialEq, Clone, Copy, Debug)]
+pub struct RadialGradientAttr {
```

Comment:
`XXXPrimIndex(u32) ` is a better name ?

Reply:
if gradient primitive index have its own type, the primitive index of color and image should have its own type too